src/samplers/mod.rs

```diff
@@ -99,6 +101,7 @@ pub trait Sampler: Sized + Send {
     /// Register all the statistics
     fn register(&self) {
         for statistic in self.sampler_config().statistics() {
+            info!("Registering statistic");
```

Comment:
This isn't a super useful log message - including the statistic name and marking it debug would be more useful. Either that or remove it entirely. Does not seem like it should be info level.

Reply:
I added it to help me understand the code flow, but should have been removed.